src/services/evm/crawl_contract_evm.service.ts

```diff
@@ -119,77 +136,49 @@ export default class CrawlSmartContractEVMService extends BullableService {
 
     const evmContractsInDB: EVMSmartContract[] = await EVMSmartContract.query()
       .select('address')
-      .whereIn('address', addresses);
+      .whereIn('address', addresses)
+      .andWhere('status', EVMSmartContract.STATUS.CREATED);
 
     const evmContractsWithAddress: any = [];
     evmContractsInDB.forEach((evmContract) => {
       evmContractsWithAddress[evmContract.address] = evmContract;
     });
+    const notFoundAddresses = addresses.filter(
+      (address: string) => !evmContractsWithAddress[address]
+    );
+    // dictionary addr->bytecode
+    const bytecodes = await this.getBytecodeContracts(notFoundAddresses);
```

Comment:
const codeByAddress: Dictionary<GetBytecodeReturnType> = await this.getBytecodeContracts(notFoundAddresses);